indexes/plugin.js

```diff
@@ -24,7 +24,7 @@ module.exports = function (
 
   const level = Level(indexPath)
   const META = '\x00'
-  const chunkSize = 512
+  const chunkSize = 2048
```

Comment:
I'm not challenging anything, just want to ask what's the back story to this, and what did you learn

Reply:
Nevermind, i just read your recent comment. :)